philipmeneghini/InstaConnect
# Email Service Registration Implementation

Email verification is now required for registration. New user is sent an email with the newly constructed email service endpoints in using Amazon SES. The user then must click on the link from the email in order to set a password and finish creating their account.

**Backend Endpoints**

-  `POST` https://localhost/7208/Email/Registration : returns a bool whether the email was successfully sent with a `UserModel` as an input parameter

![EmailServiceImplementation](https://github.com/philipmeneghini/InstaConnect/assets/63440066/fc557d21-75e3-4431-855c-e143f4cd8d5d)

**Testing**

- Make sure that verification for user information as well as password prevents the user from submitting the form.
- Check the email endpoint for expected behavior
- Check the database in order to make sure that a new user is created with a password hash
